admin_guide/backup_restore.adoc

```diff
@@ -511,6 +512,101 @@ member d266df286a41a8a4 is healthy
 xref:bringing-openshift-services-back-online[Bringing {product-title}
 Services Back Online].
 
+[[cluster-restore-etcd-quorum]]
+== Restoring etcd quorum
```

Comment:
This == section seems like an assembly in itself. Would this be better as one? I can't remember what your intentions for a Restore topic somewhere (much like the Backup topic I finished up this week), but that would help with the reuse?

Reply:
Yes, in the end, it absolutely should be its own standalone assembly.